server/util/live-loan/live-loan-fetch.js

```diff
@@ -165,15 +165,24 @@ async function fetchThemes() {
 	]);
 }
 
+async function fetchTags() {
+	return Promise.resolve([
+		{ id: 43, name: 'U.S. Black-owned Businesses' },
+		{ id: 45, name: 'Latinx/Hispanic Owned Business' },
+		{ id: 28, name: 'Repeat Borrower' }
+	]);
+}
+
 // Parse the filter string and return an array of filter arrays, e.g. [['sector', 'arts'], ['country', 'ke']]
 // Returns an empty array if the filter string does not contain any valid matches.
 const getFilterArrays = filterString => {
 	// This regex matches strings like 'sector_arts,country_ke' and captures each
 	// param name and value (for example 'sector', 'arts', 'country', 'ke' would be captured).
 	// See tests, examples, and a more detailed explanation at regexr.com/659in
-	const searchParamRegex = /(?:([a-z]+)_([a-z0-9 \\/-]+),?)+?/g;
+	const searchParamRegex = /(?:([a-z]+)_([a-z.0-9. \\/-]+),?)+?/g;
```

Comment:
The `.` only needs to be included once, and it should be escaped: `/(?:([a-z]+)_([a-z0-9 \.\\/-]+),?)+?/g`. I've updated regexr.com/659in with this change, and a test.

Reply:
Hey @emuvente, I'm getting an ESlint error with `\.`, Should I use  `\\.` instead?